docker/Dockerfile.build.aarch64-qnx

```diff
@@ -1,6 +1,7 @@
 ARG QNX_CUDA_TOOL_IMAGE_NAME
+ARG QNX_BASE_IMAGE=nvidia/cuda:10.0-devel-ubuntu16.04
```

Comment:
is it in purpose not 10.2?

Reply:
We still on DRIVE OS 5.1.0.0 what matches 10.0 (I think). I will keep it that way as there is no much hype around QNX.